### ISSUE TITLE
Certains tags font planter la "gateway" ?

### ISSUE THREAD

**Issue by bortzmeyer** [seenthis/hebergement#18]
https://seenthis.net/tag/%D1%81%D1%81%D1%81%D1%80 => 504 Gateway Time-out
The server didn't respond in time. 

Tous les autres tags que j'ai essayé" (https://seenthis.net/tag/syrie https://seenthis.net/tag/nexistepas) marchent.

**Comment by brunob** [comment]
Tiens, l'url est étrange, c'est le site qui l'a généré ? La suivante fonctionne très bien de mon côté : https://seenthis.net/tag/cccp

**Comment by bortzmeyer** [comment]
@brunob Non, c'est un copier/coller depuis mon navigateur. Celle que tu donnes n'a rien à voir (trois caractères au lieu de quatre, et caractères latins  au lieu du cyrillique)

**Comment by brunob** [comment]
Ha oui, my bad, mais même avec cccp, l'url que je cite fonctionne. Reste à vérifier que les urls qui comportent des caractères cyrilliques sont bien prises en charge par SPIP 2.1 (même si tu n'expliques pas comment tu obtiens des caractères cyrilliques en copier/coller depuis le site).

**Comment by bortzmeyer** [comment]
@brunob Comment les obtenir ? C'est facile, en cliquant sur un tag :-) (Un exemple en https://seenthis.net/messages/534314 )
